driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -238,32 +263,30 @@ fn run_solver(input_file: &str) -> Result<(), PriceFindingError> {
     Ok(())
 }
 
-fn read_output() -> std::io::Result<Value> {
+fn read_output() -> std::io::Result<String> {
     let file = File::open(format!("{}{}", RESULT_FOLDER, "06_solution_int_valid.json"))?;
-    let reader = BufReader::new(file);
-    let value = serde_json::from_reader(reader)?;
-    debug!("Solver result: {}", &value);
-    Ok(value)
+    let mut reader = BufReader::new(file);
+    let mut result = String::new();
+    reader.read_to_string(&mut result)?;
+    debug!("Solver result: {}", &result);
+    Ok(result)
 }
 
 #[cfg(test)]
 pub mod tests {
     use super::*;
     use dfusion_core::models::account_state::test_util::*;
     use dfusion_core::models::util::map_from_slice;
+    use serde_json::json;
     use std::error::Error;
     use web3::types::{H256, U256};
 
     #[test]
-
     fn test_parse_prices() {
-        let large_number: Value =
-            serde_json::from_str("340282366920938463463374607431768211457").unwrap();
-        let err = parse_price_value(&large_number).expect_err("Should fail");
-        assert_eq!(
-            err.description(),
-            "Failed to deserialize JSON Number: invalid number"
-        );
+        // 2**128 should not fit into a u128 (max value is 2**128-1)
+        let err = parse_price(&Some("340282366920938463463374607431768211457".to_owned()))
```

Comment:
Right, 2**128 is already too big. I was using 2*128 + 1, but we could decrement this string by 1 and make the last character a "6"

Reply:
Does not matter. The reason I added the comment is that I was wondering what the number is. No reason to specifically test the lowest erroring number.